crates/bevy_ecs/src/schedule/system_descriptor.rs

```diff
@@ -3,6 +3,16 @@ use crate::{
     system::{AsSystemLabel, BoxedSystem, IntoSystem},
 };
 
+/// Configures ambiguity detection for a single system.
```

Comment:
This could use a usage example; it's hard to figure out where you actually put this just from reading the docs.